data/volunteersMap.json

```diff
@@ -13,5 +13,6 @@
     "0x4213091AD181560C69F68A3a55530889CE8B052d": "ghassanmas",
     "0x6E1DDf485c8d73db8751D867067BbF3043eA98b4": "incredulous",
     "0x20aBa8cDCF10Dd5e5645d684EC99B821bC9d5C89": "IceDragon",
-    "0xD2170bd6477EB86Cb23237172764Ff3b20254Cfd": "Soumik"
+    "0xD2170bd6477EB86Cb23237172764Ff3b20254Cfd": "Soumik",
+    "0x7Bbe29411c6B55b0EcFea4efaEEc6dA3fbDB971D": "Jimmy",
```

Comment:
Remove the last comma character: `"0x7Bbe29411c6B55b0EcFea4efaEEc6dA3fbDB971D": "Jimmy"`